### PR TITLE
#5509/ Added ContactId for Contacts

### PR DESCRIPTION
**Pull request by Mihai-Bota**
Added ContactId for Contacts pojo in order to provide history search for Contacts

image

### CODE REVIEW

File: sormas-e2e-tests/src/main/java/org/sormas/e2etests/pojo/Contact.java [hunk 1]
```diff
@@ -51,4 +51,5 @@
   String contactCategory;
   String relationshipWithCase;
   String descriptionOfHowContactTookPlace;
+  String contactId;
```
Comment (BojteTamas) [inline sormas-e2e-tests/src/main/java/org/sormas/e2etests/pojo/Contact.java:54]: Please rename it to `uuid`, because you are already in `Contact` class

Comment (Mihai-Bota) [reply sormas-e2e-tests/src/main/java/org/sormas/e2etests/pojo/Contact.java:54]: will do. but it's not an UUID. i've tried to make it an UUID object but it's failing the conversion.

Comment (Mihai-Bota) [reply sormas-e2e-tests/src/main/java/org/sormas/e2etests/pojo/Contact.java]: @BojteTamas -> updated

Comment (BojteTamas) [reply sormas-e2e-tests/src/main/java/org/sormas/e2etests/pojo/Contact.java]: it's UUID, but it is created in other way than `UUID.random().toString()`

**Pull request merged**